eli5/sklearn/transform.py

```diff
@@ -25,6 +31,19 @@ def _select_names(est, in_names=None):
     return [in_names[i] for i in np.flatnonzero(mask)]
 
 
+# Scaling
+
+@transform_feature_names.register(MinMaxScaler)
+@transform_feature_names.register(StandardScaler)
+@transform_feature_names.register(MaxAbsScaler)
+@transform_feature_names.register(RobustScaler)
+def _select_scaling(est, in_names=None):
+    if in_names is None:
+        in_names = _get_feature_names(est, feature_names=in_names,
+                                      num_features=est.scale_.shape[0])
+    return [name for name in in_names]
```

Comment:
Yes, I'm converting FeatureNames instance (which comes from `_get_feature_names`) to a list. It is also a left-over from my experiments with more elaborate feature names, when you don't pass names as-is.